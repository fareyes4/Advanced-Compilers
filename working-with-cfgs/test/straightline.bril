@main {
  x: int = const 1;
  ret;
}
