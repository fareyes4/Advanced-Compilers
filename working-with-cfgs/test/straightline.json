{
  "functions": [
    {
      "name": "main",
      "instrs": [
        { "op": "const", "type": "int", "dest": "x", "value": 1 },
        { "op": "ret" }
      ]
    }
  ]
}
