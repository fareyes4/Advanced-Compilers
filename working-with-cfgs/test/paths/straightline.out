_B0 0
